src/exercises/exercise04/TruncatedText.md

```diff
@@ -0,0 +1,5 @@
+```jsx
+<TruncatedText style={{ width: 200 }}>
```

Comment:
I’d try not to use `style` on components. Since width is required, it should be a prop `width`  Also be default it should be set to `100%` for convenience.

Reply:
I think we should define a with on a parent component and/or we could use `space()` from styled-system here.